### DISCUSSION TITLE
Typo on documentation of ui.button

### DISCUSSION THREAD

**Discussion by masykur8d** [zauberzeug/nicegui#853] [General]
I am not sure if this is a typo.

The current documentation for button is:
"The color parameter excepts a Quasar color, a Tailwind color, or a CSS color."
Should it be:
"The color parameter accepts a Quasar color, a Tailwind color, or a CSS color."

<img width="508" alt="Screenshot 2023-04-28 at 14 13 56" src="https://user-images.githubusercontent.com/87178263/235060251-a5f5ef40-b180-4750-a8fb-e5c0923e599d.png">

**Comment by rodja** [comment]
Thanks for reporting. I've created #854. Will be fixed in next release.
